package-res/resources/web/pentaho/data/filter/IsIn.js

```diff
@@ -52,11 +52,18 @@ define([
    *   var filter = new IsIn("product", ["A"]);
    *   var filteredData = filter.apply(data); //filteredData.getValue(0, 0) === "A"
    * });
+   *
+   * @description Creates an IsIn filter given a property name and an array of values to be contained in the set.
+   *
+   * @param {string} property The name of the property
+   * @param {Object[]} [value] The array of corresponding property values to be contained
```

Comment:
The "value" param is not optional.

to be contained? This sounds odd.
What about something in the lines: "Comprehensive list of the values that define the set", or "array of values that belong to the set"